backend/open_webui/main.py

```diff
@@ -182,6 +184,9 @@ async def lifespan(app: FastAPI):
     docs_url="/docs" if ENV == "dev" else None, redoc_url=None, lifespan=lifespan
 )
 
+# Apply SecWeb middlewares to the app
+SecWeb(app=app, Option=get_secweb_options())
```

Comment:
Thanks for the PR, but I'd be great if we could use our own implementation instead of introducing new dependency.